### PR TITLE
New Unity Engine Cheat Sheet

### PR DESCRIPTION
**Pull request by MSDOStoevsky**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

New Cheat Sheet for the Unity game engine

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/unity_engine_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @MSDOStoevsky

### CODE REVIEW

**Comment by daxtheduck** [comment]
@MSDOStoevsky Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/unity.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
+{
+    "id": "unity_cheat_sheet",
+    "name": "Unity",
+    "description": "Key functions from the Unity game engine 5.3",
+    "metadata": {
+        "sourceName": "UnityAPI",
+        "sourceUrl": "http://docs.unity3d.com/ScriptReference/"
```
Comment (122131) [inline share/goodie/cheat_sheets/json/unity.json]: I think HTTPS it's better here 👍

**Comment by edgesince84** [comment]
Thanks Dylan!  this is pretty cool

File: share/goodie/cheat_sheets/json/unity.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        "Quaternion",
+        "Vector3",
+        "Mathf",
+        "Transform"
+    ],
+    "sections": {
+        "MonoBehaviour" : [
+            {
+                "key" : "Awake()",
+                "val": "Awake is called when the script instance is being loaded."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unity.json]: Lets remove periods at end  of val filelds

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Unity Engine Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_engine_cheat_sheet_mobile.png?2ak0OW23hX)

**Comment by alohaas** [comment]
@MSDOStoevsky this LGTM 👍 
Awesome work! I know a couple developers who are excited to use this one.

**Comment by daxtheduck** [comment]
@MSDOStoevsky this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**